nexus/reconfigurator/execution/src/dns.rs

```diff
@@ -1208,9 +1210,11 @@ mod test {
                 .into(),
                 // These are not used because we're not actually going through
                 // the planner.
+                cockroachdb_settings: &CockroachdbSettings::empty(),
                 external_ip_rows: &[],
                 service_nic_rows: &[],
                 target_nexus_zone_count: NEXUS_REDUNDANCY,
+                target_cockroachdb_cluster_version: "22.1", // YYY FIXME
```

Comment:
Yeah, a `const` somewhere was my plan. It may need to be in omicron-common, I need to look at where I've added all those comments and cross-reference it with our crate tree.